lib/task-data/schemas/install-centos.json

```diff
@@ -50,6 +50,9 @@
         "networkDevices": {
             "$ref": "types-installos.json#/definitions/NetworkDeviceArray"
         },
+        "bonds": {
```

Comment:
@sushilrai  I re-think about 'bonds' field, maybe 'bonds' is also one type of networkDevices interfaces, right?  so 'bond' could be at the same level with 'eth0' 'eth1'  under  networkDevices http://rackhd.readthedocs.io/en/latest/rackhd/install_os.html#networkdevices, right?
If I missunderstand something, please point out

Reply:
Bonds networks are defined separately for simplicity. These logical interfaces requires different configuration parameters as compared to normal physical interfaces.
Also we need to configure the bond interfaces before the normal interfaces to avoid any conflict.

@anhou please confirm if we need to include the bond interfaces in `networkDevices` then I will make necessary changes

Reply:
@sushilrai  make sense for me that it's logical device that has different configurations. +1